rftp/rate_ctrl.go

```diff
@@ -0,0 +1,96 @@
+package rftp
+
+import (
+	"sync/atomic"
+	"time"
+)
+
+type RateControl interface {
```

Comment:
In case we ever implement another `RateController`, I think we shoud move the interface to the `server.go` and use some kind of configurable factory in the server to create a new instance of that specific `RateController`.